modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -376,21 +376,83 @@ config
  config:system:set      Set a system config value
 ----
 
-You can list all configuration values with one command:
+==== config:list
+
+The `config:list` command lists all configuration values, both for your ownCloud setup, along with any apps.
 
 [source,console,subs="attributes+"]
 ----
 {occ-command-example-prefix} config:list
```

Comment:
`{occ-command-example-prefix} config:list [options] [--] [<app>] `

Reply:
Thanks for pointing that out.